sdk/src/main/java/com/hedera/hashgraph/sdk/LiveHashDeleteTransaction.java

```diff
@@ -18,23 +18,16 @@
  * by either the key of the owning account, or at least one of the keys associated to the livehash.
  */
 public final class LiveHashDeleteTransaction extends Transaction<LiveHashDeleteTransaction> {
-    private final CryptoDeleteLiveHashTransactionBody.Builder builder;
-
     @Nullable
-    AccountId accountId = null;
+    private AccountId accountId = null;
+    private byte[] hash = {};
```

Comment:
We should really be using `ByteString hash` not `byte[] hash` since `ByteString` does not copy.

I'm only going to make this comment here, but know that this should apply to all requests that store `byte[]` fields.